.travis.yml

```diff
@@ -39,6 +39,7 @@ env:
     - PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true
 
 install:
+  - set -e
```

Comment:
What does this do?

Reply:
As per the [Bash manual](https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html):

> Exit immediately if a pipeline (see Pipelines), which may consist of a single simple command (see Simple Commands), a list (see Lists), or a compound command (see Compound Commands) returns a non-zero status.

Before this, the individual scripts within these sections could fail without causing Travis to fail the build.